oe_corporate_site_info.module

```diff
@@ -17,6 +19,24 @@ function oe_corporate_site_info_form_system_site_information_settings_alter(&$fo
   $corporate_site_info = \Drupal::config('oe_corporate_site_info.settings');
   $skos_concept_storage = \Drupal::entityTypeManager()->getStorage('skos_concept');
 
+  $accessibility_uri = $corporate_site_info->get('accessibility') ?? '';
+  $form['accessibility'] = [
+    '#type' => 'textfield',
+    '#title' => t('Accessibility statement'),
+    '#required' => TRUE,
+    '#description' => t('Link to the accessibility statement of your website'),
+    '#process' => \Drupal::service('element_info')->getInfoProperty('entity_autocomplete', '#process'),
+    '#default_value' => _oe_corporate_site_info_get_privacy_default($accessibility_uri),
```

Comment:
You are getting a default value here, so please reflect this in the function name.